x-pack/test/functional/apps/ml/data_visualizer/index_data_visualizer_filters.ts

```diff
@@ -0,0 +1,143 @@
+          await ml.dataVisualizerIndexBased.assertFilterBarFilterContent({
+            key: filter.key,
+            value: filter.value,
+            enabled: true,
+            pinned: false,
+            negated: false,
+          });
+        }
+      }
+
+      await ml.dataVisualizerIndexBased.assertFilterBarFilterContent(PINNED_FILTER);
+    });
+
+    it(`retains pinned filters to other plugins`, async () => {
+      await ml.testExecution.logTestStep(`${testData.suiteTitle} navigates to ML`);
+      await ml.navigation.navigateToDataVisualizer();
+
+      await ml.testExecution.logTestStep(
+        `${testData.suiteTitle} loads the saved search selection page`
+      );
+      await ml.dataVisualizer.navigateToIndexPatternSelection();
+
+      await ml.testExecution.logTestStep(
+        `${testData.suiteTitle} loads the index data visualizer page`
+      );
+      await ml.jobSourceSelection.selectSourceForIndexBasedDataVisualizer(
+        testData.sourceIndexOrSavedSearch
+      );
+
+      await ml.testExecution.logTestStep(`${testData.suiteTitle} loads data for full time range`);
+      await ml.dataVisualizerIndexBased.clickUseFullDataButton(
+        testData.expected.totalDocCountFormatted
+      );
+      await PageObjects.header.waitUntilLoadingHasFinished();
+
+      await ml.testExecution.logTestStep(`${testData.suiteTitle} adds a pinned filter`);
+      await filterBar.addFilter(PINNED_FILTER.key, 'is', PINNED_FILTER.value);
+      await filterBar.toggleFilterPinned(PINNED_FILTER.key);
+      await PageObjects.header.waitUntilLoadingHasFinished();
+
+      await ml.testExecution.logTestStep(`${testData.suiteTitle} navigates to Discover`);
+      await ml.navigation.navigateToDiscoverViaAppsMenu();
+      await PageObjects.header.waitUntilLoadingHasFinished();
+      await ml.dataVisualizerIndexBased.assertFilterBarFilterContent(PINNED_FILTER);
+    });
+  }
+  describe('data visualizer with pinned global filters', function () {
+    before(async function () {
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/ml/farequote');
+      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/ml/module_sample_logs');
+      await ml.testResources.createIndexPatternIfNeeded('ft_farequote', '@timestamp');
+      await ml.testResources.createSavedSearchFarequoteFilterAndLuceneIfNeeded();
+      await ml.testResources.createSavedSearchFarequoteFilterAndKueryIfNeeded();
+
+      await ml.securityUI.loginAsMlPowerUser();
+    });
+
+    after(async function () {
+      await ml.testResources.deleteSavedSearches();
+      await ml.testResources.deleteIndexPatternByTitle('ft_farequote');
+    });
+
+    describe(`with ${farequoteDataViewTestData.suiteTitle}`, function () {
+      runTests(farequoteDataViewTestData);
+    });
+
+    describe(`with ${farequoteLuceneFiltersSearchTestData.suiteTitle}`, function () {
+      runTests(farequoteKQLFiltersSearchTestData);
```

Comment:
I think this should rather be
```suggestion
      runTests(farequoteLuceneFiltersSearchTestData);
```

Reply:
Updated here [farequoteLuceneFiltersSearchTestData](https://github.com/elastic/kibana/pull/138539/commits/3d51c9b472d17f6b869b5eb231b658f247090b26)